config/webpack.config.js

```diff
@@ -23,12 +24,12 @@ const entryFiles = {
   'hca-rjsf': './src/js/hca-rjsf/hca-rjsf-entry.jsx',
   'health-records': './src/js/health-records/health-records-entry.jsx',
   messaging: './src/js/messaging/messaging-entry.jsx',
+  pensions: './src/js/pensions/pensions-entry.jsx',
+  'post-911-gibs': './src/js/post-911-gibs/post-911-gibs-entry.jsx',
```

Comment:
Yeah, good point--I like `post-911-gib-status` better. Will update tomorrow and merge.